app/uk/gov/hmrc/excisemovementcontrolsystemapi/connectors/util/EISHttpReader.scala

```diff
@@ -52,10 +52,10 @@ class EISHttpReader(
     logger.warn(EISErrorMessage(createdDateTime, consignorId, message, correlationId, MessageTypes.IE815Message))
 
     status match {
-      case BAD_REQUEST => BadRequest(message)
-      case NOT_FOUND => NotFound(message)
-      case SERVICE_UNAVAILABLE => ServiceUnavailable(message)
-      case _ => InternalServerError(message)
+      case BAD_REQUEST => BadRequest(message).as("application/json")
```

Comment:
ah so this is partially me fogetting things. Will retest

Basically, I noticed on Postman i was getting eis errors back as text not json and this fixed it. But then I forgot to do it for the other cases!

I will make sure I'm setting Accepted and if that resolves it, I'll remove this. Else I'll make it consistent.

Reply:
Ok so I pass in Accept header as application/json and it still returns it as a text Content-Type without the .as(application/json) so I'm gonna leave that in for now. I think its because it interprets the return type from the body type which is passed to us as a string. But we know EIS will give us JSON here so I don't think it will cause any issues.